packages/core/index.js

```diff
@@ -89,7 +89,8 @@ const trifid = async (config, additionalPlugins = {}) => {
     trustProxy: true,
     ...serverOptions,
   })
-  await server.register(fastifyCompress)
+  // Register fastifyCompress and add custom compressible mime-types
+  await server.register(fastifyCompress, { customTypes: /n-triples$/ })
```

Comment:
Is it the intention to only compress n-triples?

Reply:
is trifid ever used to serve uncompressible content?
if that's not the case, it might be useful to include `/.*/` WDYT?

Reply:
or even better to avoid overriding uncompressible `static-assets` in case such assets exist:
- `/(n-triples|n-quads|trig)$/`